hbase-build-configuration/pom.xml

```diff
@@ -68,11 +68,6 @@
       <activation>
         <activeByDefault>false</activeByDefault>
       </activation>
-      <properties>
```

Comment:
Since this check is run by precommit testing only, I'm fine with bumping this check over to JDK11. Please make sure that test-patch is calling it from the jdk11 incantation instead of the jdk8 incantation.